cryptographic_estimators/UOVEstimator/uov_problem.py

```diff
@@ -24,23 +24,20 @@
 
 
 class UOVProblem(BaseProblem):
-    """
-    Construct an instance of UOVProblem.
-
-    INPUT:
-
-    - ``n`` -- number of variables
-    - ``m`` -- number of polynomials
-    - ``q`` -- order of the finite field
-    - ``theta`` -- exponent of the conversion factor (default: 2)
-        - If ``0 <= theta <= 2``, every multiplication in GF(q) is counted as `log2(q) ^ theta` binary operation.
-        - If ``theta = None``, every multiplication in GF(q) is counted as `2 * log2(q) ^ 2 + log2(q)` binary operation.
-    - ``cost_one_hash`` -- bit complexity of computing one hash value (default: 17)
-    - ``memory_bound`` -- maximum allowed memory to use for solving the problem (default: inf)
-
-    """
-
     def __init__(self, n: int, m: int, q: int, **kwargs):
+        """Construct an instance of UOVProblem.
+
+        Args:
+            n (int): Number of variables
+            m (int): Number of polynomials
+            q (int): Order of the finite field
+            **kwargs: Additional keyword arguments
+                theta (float, optional): Exponent of the conversion factor. Defaults to 2.
+                    If 0 <= theta <= 2, every multiplication in GF(q) is counted as log2(q) ^ theta binary operation.
+                    If theta = None, every multiplication in GF(q) is counted as 2 * log2(q) ^ 2 + log2(q) binary operation.
+                cost_one_hash (int, optional): Bit complexity of computing one hash value. Defaults to 17.
+                memory_bound (float, optional): Maximum allowed memory to use for solving the problem. Defaults to inf.
+        """
```

Comment:
wrong indentation, idk if this has any consequences on the docs

Reply:
to be fixed in  #198

Reply:
Nothing to fix. Indentation is correctly parsed by our custom Sphinx extension (see https://github.com/Crypto-TII/CryptographicEstimators/pull/211).